scripts/on-rpi-boot.sh

```diff
@@ -21,33 +26,34 @@ Pin-Priority: -10
 EOF
 fi
 
-if [ "$(grep -c "http://archive.raspbian.org/raspbian wheezy" /etc/apt/sources.list)" = 0 ]
+if ! egrep -q '^\s*http://archive\.raspbian\.org/raspbian\s*wheezy' /etc/apt/sources.list
 then
-  echo "deb http://archive.raspbian.org/raspbian wheezy main contrib non-free rpi" >> /etc/apt/sources.list
+  echo 'deb http://archive\.raspbian\.org/raspbian wheezy main contrib non-free rpi' >> /etc/apt/sources.list
 fi
```

Comment:
That one is definitely a mistake. I meant to only escape the periods in the regex.